hardware/src/base-station-sketch/example-pycmdmessenger-sketch.ino

```diff
@@ -0,0 +1,70 @@
+/*
+  Example sketch utilising PyCmdMessenger Library
+
+  This sketch contains several basic use cases of the PyCmdMessenger
+  library.
+
+  TODO Requires full compatiability testing once hardware base station is complete.
+
+  Created 7 September 2016
+  By Marcel van Workum
+
+  Modified 7 September 2016
+  By Marcel van Workum
+*/
+
+#include "CmdMessenger.h"
+
+/* Define available CmdMessenger commands */
+enum {
+    current_status,
+    status_is,
+    think_for_me,
```

Comment:
These are kind of oddly named. I get that it's an example but it doesn't relate to rocket things